### PR TITLE
Success

### PR DESCRIPTION
**Pull request by andrzejkukuryk**
add next screen
change screen display way

### CODE REVIEW

File: src/store/appControl/appControlSlice.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { createSlice } from "@reduxjs/toolkit";
+import type { PayloadAction } from "@reduxjs/toolkit";
+
+export type currentScene = "splash" | "animation" | "creator" | "success";
```
Comment (anna-morawska) [inline src/store/appControl/appControlSlice.tsx]: the convention is to have uppercased types names

File: src/scenes/success/success.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+import { useDispatch } from "react-redux";
+import {
+  updateCurrentScene,
+  updateStartAnimation,
+} from "../../store/appControl/appControlSlice";
+
+export function Success() {
+  const dispatch = useDispatch();
+
+  const circleBigPositions: PositionCircleBigInfo[] = [
```
Comment (anna-morawska) [inline src/scenes/success/success.tsx]: it' better to define constants outside of the component body

File: src/scenes/sceneChanger.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import { currentSceneSelector } from "../store/appControl/selectors";
+
+interface SceneChangerProps {
+  children: ReactNode;
+}
+
+export function SceneChanger({ children }: SceneChangerProps) {
+  const scene = useSelector(currentSceneSelector);
+
+  const arrayOfChildren = React.Children.toArray(children);
```
Comment (anna-morawska) [inline src/scenes/sceneChanger.tsx:14]: why do you do it like this? Quite uncommon approach

File: src/store/appControl/appControlSlice.tsx (hunk 1 above)
Comment (andrzejkukuryk) [reply src/store/appControl/appControlSlice.tsx]: fixed

File: src/scenes/success/success.tsx (hunk 2 above)
Comment (andrzejkukuryk) [reply src/scenes/success/success.tsx]: fixed

File: src/scenes/sceneChanger.tsx (hunk 3 above)
Comment (andrzejkukuryk) [reply src/scenes/sceneChanger.tsx:14]: I don't know how to choose only one child to display from all passed to the component. This:  `return children[0]` doesn't work.

**Pull request merged**